dcicutils/deployment_utils.py

```diff
@@ -66,15 +74,10 @@ def boolean_setting(settings, key, default=None):
 
 class EBDeployer:
 
+    # Options specifically related to our deployment
     S3_BUCKET = 'dcic-application-versions'
     EB_APPLICATION = '4dn-web'  # XXX: will need to change if this changes -Will
```

Comment:
Ha, it will sound funny to say it this way, but then make a constant of it.

I remember seeing such advice where someone was using 3.14159 in code and was told to make it be a constant in case it needs to change. Surprisingly, ordinary reasons to want to change the value of PI com up all the time, even other than [the stupid reasons](http://scienticity.net/wiki/Legislating_the_Value_of_Pi).